### PR TITLE
[MIG][12.0] product_cost_security: Migration to v12.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT20172
@carlosdauden @chienandalu review please

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested. :+1:

File: product_cost_security/__manifest__.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 {
     'name': 'Product Cost Security',
     'summary': 'Product cost security restriction view',
-    'version': '11.0.1.0.0',
-    'development_status': 'Beta',
+    'version': '12.0.1.0.0',
+    'development_status': 'Production/Stable',
```
Comment (pedrobaeza) [inline product_cost_security/__manifest__.py:7]: You need to declare `maintainers` if promoting to stable

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
Changes done!!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**